cmp/grid/Grid.scss

```diff
@@ -192,3 +192,22 @@
 .xh-grid-tooltip {
   position: absolute;
 }
+
+.xh-grid-autosize-canvas {
+  position: absolute;
+  display: none;
+}
+
+.xh-grid-autosize-cell {
+  top: var(--xh-grid-tree-indent-px);
+  padding: 0 var(--xh-grid-cell-lr-pad-px);
+  font-family: var(--xh-grid-font-family);
+  font-size: var(--xh-grid-font-size-px);
+  -webkit-font-smoothing: subpixel-antialiased;
```

Comment:
I would be pretty surprised if this font-smoothing rule made a difference (although certainly could be wrong).  I would vote for removing it in the interest of simplicity, unless we have reason to believe it's important.